packages/skeleton/src/lib/components/ListBox/ListBoxItem.svelte

```diff
@@ -17,19 +17,23 @@
 	export let name: string;
 	/** Set the input's value. */
 	export let value: any;
+	/** Disables selection of the item. */
+	export let disabled: boolean = false;
 
 	// Context
+	export let disabledItems: boolean = getContext('disabledItems');
 	export let multiple: string = getContext('multiple');
 	export let rounded: CssClasses = getContext('rounded');
 	export let active: CssClasses = getContext('active');
+	export let activeDisabled: CssClasses = getContext('activeDisabled');
 	export let hover: CssClasses = getContext('hover');
 	export let padding: CssClasses = getContext('padding');
 	export let regionLead: CssClasses = getContext('regionLead');
 	export let regionDefault: CssClasses = getContext('regionDefault');
 	export let regionTrail: CssClasses = getContext('regionTrail');
 
 	// Classes
-	const cBase = 'cursor-pointer -outline-offset-[3px]';
+	const cBase = '-outline-offset-[3px]';
```

Comment:
let's re-add `cursor-pointer` here and override it using the disabled style instead.

Reply:
I've re added it but I'm not sure how can i override the disabled style. Could you elaborate more on that?

Reply:
look at https://github.com/skeletonlabs/skeleton/blob/c6a4ddc02d2883e2ff0c350cc5f02f3b69d0d9ce/packages/skeleton/src/lib/components/TreeView/TreeViewItem.svelte#L320C33-L320C51
for how we handled this in other components :+1:

Reply:
Ok, let me know if i'm doing this correctly. I've removed entirely the possibility for styling the active disabled items and replaced it with opacity which seems better, using the cDisabled approach.